python_modules/dagster-graphql/dagster_graphql/schema/config_types.py

```diff
@@ -107,6 +109,32 @@ def resolve_given_name(self, _):
         return self._config_type_snap.given_name
 
 
+class GrapheneMapConfigType(ConfigTypeMixin, graphene.ObjectType):
+
+    key_type = graphene.Field(graphene.NonNull(GrapheneConfigType))
+    of_type = graphene.Field(graphene.NonNull(GrapheneConfigType))
```

Comment:
`value_type` ? `of_type` reads weird to me

Reply:
updated